apps/astarte_data_updater_plant/lib/astarte_data_updater_plant_web/metrics/data_updater_plant_instrumenter.ex

```diff
@@ -0,0 +1,102 @@
+#
+# This file is part of Astarte.
+#
+# Copyright 2020 Ispirata Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+defmodule Astarte.DataUpdaterPlantWeb.Metrics.DataUpdaterPlantInstrumenter do
+  use Prometheus.Metric
+
+  @connection_counter_specs [
+    name: :connected_devices_total,
+    help: "DataUpdaterPlant devices connection count",
+    labels: [:realm]
+  ]
+
+  @disconnection_counter_specs [
+    name: :disconnected_devices_total,
+    help: "DataUpdaterPlant devices disconnection count",
+    labels: [:realm]
+  ]
+
+  @connected_devices_specs [
+    name: :connected_devices,
+    help: "DataUpdaterPlant count of devices currently connected",
+    labels: [:realm]
+  ]
+
+  @discarded_messages_spec [
+    name: :discarded_messages_total,
```

Comment:
I'd drop the `_total`

Reply:
By convention, the names of Counters are suffixed by `_total`